app/mailers/code_buddy_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CodeBuddyMailer < ActionMailer::Base
+  default from: "from@example.com"
+
+  def review_posted(current_user, review)
```

Comment:
Unused method argument - `review`. If it's necessary, use `_` or `_review` as an argument name to indicate that it won't be used.